mkdocs/docs/protocol/non-merklized.md

```diff
@@ -0,0 +1,126 @@
+# Non-merklized credentials
+
+## Motivation
+
+While Iden3 protocol allows to have a root of  merklized JSON-LD document for proving the inclusion of the credential entries in the tree, that increases level of privacy and flexibility for the credential data there is also a need to issue a core claim with a data itself. This is useful as for onchain issuers that can’t work with ld schemas in smart contracts and also for use case when the actual data must be saved, e.g. Auth BJJ credential public key representation.
```

Comment:
While Iden3 protocol allows to "merklize" the whole JSON-LD document (transform the document into key-value pairs and put them into a sparse merkle tree for later proving of credential field value in the tree), which removes limits on amount and structure of the credential data, there's still a need to issue a core claim with the data itself. This is particularly useful for onchain issuers, which can’t work with json-ld schemas in smart contracts (for many reasons), and also for use cases when the actual data must be saved for cheaper access inside circuits (e.g. Auth BJJ credential public key representation).

Reply:
fixed